test/extended/util/network/network.go

```diff
@@ -0,0 +1,47 @@
+package network
```

Comment:
Is this likely to ever be used by tests outside of `test/extended/network`? If not it should probably just go into `test/extended/network`. (There are already utils there.)

Reply:
moved to network utils